### PR TITLE
Misc improvements

### LINKED ISSUE #30: Add apidocs
Add Django APIdocs

### LINKED ISSUE #41: Fix incorrect courses display bug
Currently, the collection action only gets 11,000 entries from the `api/courses/` endpoint. There are a total of 11,069 entries collected by the scraper.

This falls in line with issue #25

### LINKED ISSUE #44: Add page title and favicon
Currently, the page title is React App and there is no favicon. We don't specifically need an icon, but we should at least have a page title.

### PR DESCRIPTION
**Pull request by Blu3spirits**
This PR should close out #44 #41 #30.

We should really figure out #25 as we'll keep running into #41-like issues as tech adds more or fewer data entries.

### CODE REVIEW

**Comment by Blu3spirits** [comment]
Adding all those tags looked like clown vomit.

File: webapp/src/actions.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
 } from './action-types';
 
 export const getCourses = () => createAction({
-  endpoint: `${ENDPOINT}/api/courses/?limit=11000`,
+  endpoint: `${ENDPOINT}/api/courses/?limit=12000`,
```
Comment (codetheweb) [inline webapp/src/actions.js:10]: We could just set `limit=1000000` until we figure out #25 if you want, although I don't think it matters too much.

File: course_gather/settings/dev_settings.py [hunk 2]
```diff
@@ -14,6 +14,10 @@
     'debug_toolbar.middleware.DebugToolbarMiddleware',
 )
 
+REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] += (  # noqa: F405
```
Comment (codetheweb) [inline course_gather/settings/dev_settings.py:17]: Do we want to enable the docs for prod as well (or just put in base settings)?

File: webapp/src/actions.js (hunk 1 above)
Comment (Blu3spirits) [reply webapp/src/actions.js:10]: Strangely, it does matter in the browser. Times go up by ~400ms with absurdly high ?limit= parameters. CLI, with curl, experiences no such odd changes.

File: course_gather/settings/dev_settings.py (hunk 2 above)
Comment (Blu3spirits) [reply course_gather/settings/dev_settings.py:17]: This specific line change is for the Django browsable API, not the API docs.
API docs are enabled in prod, via the `installed_apps` and in `urls.py`

Comment (codetheweb) [reply course_gather/settings/dev_settings.py:17]: Took a look at the docs and it makes more sense now, my bad.

File: webapp/src/actions.js (hunk 1 above)
Comment (codetheweb) [reply webapp/src/actions.js:10]: 🤔 it looks pretty consistent to me:
Screen Shot 2020-05-26 at 6 26 00 PM

Again, doesn't really matter if you just wanna get this merged in tonight but I dunno why that'd be happening for you.

**Review by codetheweb — APPROVED** [review]

File: webapp/src/actions.js (hunk 1 above)
Comment (Blu3spirits) [reply webapp/src/actions.js:10]: Weird, anyway for now I'm content to just update it to the next highest number.

**Pull request merged**